WooCommerce/WooCommerceScreenshots/Screens/Login/LoginPasswordScreen.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+import XCTest
+
+private struct ElementStringIDs {
+    static let passwordTextField = "Password"
+    static let loginButton = "Password Next Button"
+    static let errorLabel = "pswdErrorLabel"
+}
+
+class LoginPasswordScreen: BaseScreen {
+    let passwordTextField: XCUIElement
+    let loginButton: XCUIElement
+
+    init() {
+        passwordTextField = XCUIApplication().secureTextFields[ElementStringIDs.passwordTextField]
+        loginButton = XCUIApplication().buttons[ElementStringIDs.loginButton]
+        super.init(element: passwordTextField)
+    }
+
+    func proceedWith(password: String) -> LoginEpilogueScreen {
+        _ = tryProceed(password: password)
+
+        return LoginEpilogueScreen()
+    }
+
+    func tryProceed(password: String) -> LoginPasswordScreen {
+        let tmp = UIPasteboard.general.string
+        UIPasteboard.general.string = password
+        passwordTextField.tap()
+        XCUIApplication().menuItems.firstMatch.tap()
```

Comment:
This method for entering the password is a little flaky for me. It seems to work most of the time, but sometimes (I think fairly consistently when I start with the simulator closed and let the test launch it) the tap fails to bring up the contextual menu and the test fails here.

What do you think of using the method that we use in the WordPress app where we type the password instead of pasting it?

Reply:
Great question! 😄 

This is what I had initially, but it didn't work! I found it would _never_ enter the password. I saw the flakiness as well in the "tap to enter password" thing, but given that the screenshots code retries automatically, it never failed to run the suite (whereas the original implementation never succeeded). 

It's possible this is something weird on my machine, so I'd actually _prefer_ to use the same implementation as WPiOS if we can! This is something we could try to debug further before merging for sure.

Reply:
Ok – so I synced the implementation from WPiOS and now it works 🤯

Tested on a simulated iPhone X on iOS 13, 12.2, and 11.4.

Reply:
Magic! 😄 This implementation is also working well on my end (tested on a simulated iPad Pro on iOS 13 as well).